static/app/views/explore/tables/aggregateColumnEditorModal.tsx

```diff
@@ -142,6 +196,139 @@ function ColumnEditorRow({canDelete, column, onColumnDelete}: ColumnEditorRowPro
   );
 }
 
+interface GroupBySelectorProps {
+  groupBy: GroupBy;
+  groupBys: string[];
+  onChange: (groupBy: GroupBy) => void;
+  stringTags: TagCollection;
+}
+
+function GroupBySelector({
+  groupBy,
+  groupBys,
+  onChange,
+  stringTags,
+}: GroupBySelectorProps) {
+  const options: Array<SelectOption<string>> = useGroupByFields({
+    groupBys,
+    tags: stringTags,
+  });
+
+  const label = useMemo(() => {
+    const tag = options.find(option => option.value === groupBy.groupBy);
+    return <TriggerLabel>{tag?.label ?? t('None')}</TriggerLabel>;
```

Comment:
Is `None` supposed to render in place of the hyphen we use? Since the `useGroupByFields` adds in the empty option this doesn't get hit, nor does it for tags that are missing from the options because I think they're inserted if they're missing from the current tags.

Just curious, if this is a "worst-case" fallback, then it's fine

Reply:
Good point, it should fall back to `groupBy.groupBy` instead as that's the selected value.